operations/research-and-development/product/release-process/release-overview.md

```diff
@@ -89,10 +152,22 @@ Mattermost numbers stable releases in the following format:
 
   - A: See instructions here: https://developers.mattermost.com/internal/mobile-build-process/.
 
-**Q: How does updating dependencies work?**
-
-  - A: Dependency updates will only occur in feature releases, unless they contain security fixes.
-
 **Q: What is the process for community PRs?**
 
   - A: Review, merge, and cherry-pick.
+
+**Q: Will RC testing remain, or will RC testing process change?**
+
+  - There will be some manual RC testing until all release tests are automated.
+
+**Q: Are mobile and desktop included in the cloud first strategy?**
```

Comment:
```suggestion
**Q: Are mobile and desktop included in the "cloud first" strategy?**
```

Reply:
I've left this as "cloud" because it's how we refer to this strategy internally.